### ISSUE TITLE
Add word to hyphenation dictionary

### ISSUE THREAD

**Issue by bountonw** [bountonw/translate#248]
@mattleff 

Please add this word to the hyphenation dictionary and help me to know what the rules are so that I can update the dictionary on my own. The dictionary should trigger new LaTeX pdfs and InDesign files.

`เธสะ~โล~~นิกา`

CC @kenton-r

**Comment by mattleff** [comment]
@bountonw Per the message from Kenton in the chat group (4/18/24 3:31PM) I'm thinking we aren't going to be dealing with hyphenation (at least for LBF):

> ...
> We will not be using a hyphenation list as HarfBuzz does not utilize the word lists stored in Hunspell. But I believe that hyphens will not be needed.

Is this correct, @kenton-r?

**Comment by kenton-r** [comment]
Yes, any hyphenation will have to be manually added, because Hunspell is unaware of the line breaks HarfBuzz creates with clusters. But using forced line breaks are preferred if the word is not too big.
